rental.rb

```diff
@@ -0,0 +1,14 @@
+# this class represents the rental
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Getters and Setters for book and person but you are only setting the Getter, it is necessary for the two to be writeable.

Reply:
Thank you @Olamarx. Done!